lib/graphql/query/input_validation_result.rb

```diff
@@ -4,12 +4,20 @@ class Query
     class InputValidationResult
       attr_accessor :problems
 
+      def initialize(schema)
+        @schema = schema
+        @problems = []
+      end
+
       def valid?
-        @problems.nil?
+        @problems.empty?
+      end
+
+      def too_many_errors?
+        @problems.size >= @schema.max_validation_errors
```

Comment:
IIRC, this object is initialized _for each input_ that's validated. Many validations don't use this at all. For example, `fieldsExistOnType` runs a lot of checks, but never uses this class. 

Is this change intended to cover validations like that, or only validations on provided inputs?

Reply:
I was intending the change to `lib/graphql/static_validation/validator.rb` to cover all the other validators, but I'm not sure it did.